spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+from cerberus import Validator as cerberus_validator
+from spidermon.contrib.validation.validator import Validator
+
+from .translator import CerberusMessageTranslator
+
+
+class CerberusValidator(Validator):
+    default_translator = CerberusMessageTranslator()
+    name = "Cerberus"
+
+    def __init__(self, schema, translator=None, use_default_translator=True):
+        super(CerberusValidator, self).__init__(
+            translator=translator, use_default_translator=use_default_translator
```

Comment:
Each argument could go in one line.
Just readability tips :wink: 
Also, since these are not used by this class, only the super class, they could be `*args` and **kwargs`

Reply:
Thanks, Julio, really need to follow on the best practices. I just formatted my code with `Black` and submitted it. Will fix it.
I have read about `*args` and `**kwargs`, let me try to implement what you are saying and get back to you for a re-review if I did it right.

Reply:
I read about `*args` and `**kwargs` and I think, these won't work out. 
1. Since we would need them to initialize the default_translator variable quite explicitly. 
2. I tried it and it doesn't work out. 
Resolving.